src/query/xasl_cache.c

```diff
@@ -2353,6 +2353,7 @@ xcache_check_recompilation_threshold (THREAD_ENTRY * thread_p, XASL_CACHE_ENTRY
   CLS_INFO *cls_info_p = NULL;
   int npages;
   bool recompile = false;
+  bool use_stat_estimation = prm_get_bool_value (PRM_ID_USE_STAT_ESTIMATION);
```

Comment:
The use_stat_estimation variable can be removed by modifying the if() statement below.
```suggestion  
```